entropylab/results_backend/sqlalchemy/tests/test_db.py

```diff
@@ -35,3 +35,31 @@ def test_get_last_result_of_experiment_works_with_db(initialized_project_dir_pat
     actual = db.get_last_result_of_experiment(1)
     # assert
     assert actual.data == "to db"
+
```

Comment:
what are these tests?

Reply:
These are two tests that target get_last_result_of_experiment(). One tests it with the default flow that fetches from HDF5. The other tests the backup flow (`enable_hdf5_storage=False) that fetches it from the old DB table.
I'll rename them now to make the difference clearer.